### ISSUE TITLE
Valorant not showing as installed on the last update of GOG

### ISSUE THREAD

**Issue by BanCrash** [urwrstkn8mare/gog-riot-integration#10]
Hello! There was a GOG update on 1 october and as result, Valorant is not showing anymore as installed.

I tried deleting the plugin and reinstalling it but it only shows Valorant as installed on the first running. After closing GOG and opening it again, it appears again as uninstalled:

![image](https://user-images.githubusercontent.com/11770760/95250499-77fa5b00-081a-11eb-87ae-0fc1c5b3538d.png)

Regards and thanks.

**Comment by urwrstkn8mare** [comment]
Thanks! I will get on this ASAP.

**Comment by BanCrash** [comment]
Hello, thanks you! Btw, I said it was for the update, but maybe it's just an assumption, I don't know for sure if that's the case.

BTW, here I let you the riot plugin log.
[plugin-riot-b4c0c2f5-23dc-4326-aa65-e9fabef61a3b.log](https://github.com/urwrstkn8mare/galaxy-riot-integration/files/5337645/plugin-riot-b4c0c2f5-23dc-4326-aa65-e9fabef61a3b.log)

**Comment by urwrstkn8mare** [comment]
@BanCrash That's really odd because I have the update and Valorant shows as installed for me. But what is really odd is that the logs say it's installed. Here is a snippet from the logs.

```
2020-10-07 00:31:30,064 - plugin - DEBUG - self.status: {'bacon': <LocalGameState.None_: 0>, 'league_of_legends': <LocalGameState.None_: 0>, 'valorant': <LocalGameState.Installed: 1>}
```

Have you tried disconnecting and reconnecting the plugin? I believe the playtime data show be safe.

**Comment by BanCrash** [comment]
Yes, I tried, and to appear again I have to do this, otherwise it will not appear:

-Disconnect the plugin.
-Close and open again GOG.
-Connect the plugin.

With these steps, it will appear as installed on that launch, but on the next launch it will appear again as uninstalled. If I don't close GOG after disconnecting the plugin it will not appear after connect the plugin again though.

I tried v0.2.3 and v0.2.2 but none of them solves the isue.

**Comment by urwrstkn8mare** [comment]
@BanCrash Please see if this fixes your issue: https://github.com/urwrstkn8mare/galaxy-riot-integration/releases/tag/v0.2.4

**Comment by BanCrash** [comment]
I see a little improvement, one time I get it as installed, but only 1 of 3. I don't know what happens... I reinstalled again two times the plugin to see if that solves the issue but appears not.

**Comment by fl4shback** [comment]
@urwrstkn8mare 
Did a bit of research on this one.

It seems like Galaxy only triggers the get_local_games() function on the first plugin launch a never seems to trigger it again until a full login/logout.

You can see that the debug message "Getting local games" never seems to appear in the logs after the first launch of the plugin.

I wasn't able to find the reason for this..


[plugin-riot-b4c0c2f5-23dc-4326-aa65-e9fabef61a3b.log](https://github.com/urwrstkn8mare/galaxy-riot-integration/files/5792712/plugin-riot-b4c0c2f5-23dc-4326-aa65-e9fabef61a3b.log)

[EDIT]
It seems like the get_owned_games() function is impacted by this too

[EDIT2]
Really interesting. I let galaxy run untouched for almost 2 hours and noticed that the games appeared as installed.
After logs examination, it seems that for some reason, the plugin  is skipping the first wave of imports when galaxy start.
After one hour, Galaxy runs a new wave of imports and this time it works...

Search for string "import installed games" in this new log file. (battlenet plugin seems to suffer from this too)
[GalaxyClient.log](https://github.com/urwrstkn8mare/galaxy-riot-integration/files/5798331/GalaxyClient.log)

**Comment by urwrstkn8mare** [comment]
Just a quick update, I'm low on time to work on this so I haven't been able to find a solution to this. The only fix I can offer is when this happens, please disconnect and reconnect the plugin, the playtime info should carry over automatically. 

Thanks for the feedback.
